desc/profiles.py

```diff
@@ -214,13 +653,9 @@ def set_params(self, l, a=None):
         for ll, aa in zip(l, a):
             idx = self.basis.get_idx(ll, 0, 0)
             if aa is not None:
-                self.params[idx] = aa
+                self.params = put(self.params, idx, aa)
 
-    def get_idx(self, l):
```

Comment:
got rid of this just because Profile.basis.get_idx serves same purpose and is more general ?

Reply:
yeah basically